vscode-dotnet-runtime-extension/.vscode/launch.json

```diff
@@ -5,7 +5,7 @@
 {
 	"version": "0.2.0",
 	"configurations": [{
-			"name": "Run Acquisition Extension",
+			"name": "Run .NET Runtime Acquisition Extension",
```

Comment:
Hmm, do you think your typical workflow in this repo will consist of you opening each individual folder in code or opening the workspace directory in code? Asking because I imagine there will be a lot of shared code among the two extensions in regards to acquisition.

Reply:
Yes, I imagine we'll be opening the workspace directly, which will include both extensions. This renaming ensures that, when there's another extension with its own launch tasks, we don't get confused between SDK and runtime tests.

Reply:
Can you have multiple .vscode folders/launch.json's in sub-folders? Haven't ever tried 😄

Reply:
Yup, that's how it works right now with the sample extension vs the runtime extension.

Reply:
Out of curiosity then, does it have a "smart" `{workspaceFolder}` understanding based on where the .vscode folder is?

Reply:
Yes, for example: 
![Screenshot (2)](https://user-images.githubusercontent.com/16010855/106955699-6f436400-66ea-11eb-896d-50a8de1379a0.png)